deploy/cloud-node-controller-role.yaml

```diff
@@ -5,6 +5,14 @@ metadata:
     addonmanager.kubernetes.io/mode: Reconcile
   name: system:cloud-controller-manager
 rules:
+- apiGroups:
+  - networking.gke.io
+  resources:
+  - gkenetworkparamset
+  - gkenetworkparamset/status
+  verbs:
+  - update
```

Comment:
informers need
```
  - list
  - watch
```

Reply:
on line 85 of this file, these verbs are given to all resources
```
- apiGroups:
  - "*"
  resources:
  - "*"
  verbs:
  - list
  - watch
```

Reply:
we should change that in the future 👍